src/Tools/LeanCode.ContractsGenerator/GeneratorConfiguration.cs

```diff
@@ -26,7 +26,7 @@ public static async Task<List<GeneratorConfiguration>> GetConfigurations(string
             return await JsonSerializer.DeserializeAsync<List<GeneratorConfiguration>>(stream);
         }
 
-        public static List<GeneratorConfiguration> DeserializeConfigurationsFromString(string cfg)
+        public static List<GeneratorConfiguration>? DeserializeConfigurationsFromString(string cfg)
```

Comment:
I'd probably consider coercing `null`s to empty lists with `?? new List<GeneratorConfiguration> { }` instead.